src/lazy/errors/loader-error.ts

```diff
@@ -0,0 +1,16 @@
+export class LoaderError extends Error {
+  public readonly nativeError: Error;
+
+  constructor(id: string, error: Error) {
+    super();
+    this.message = `Failed to load module ${id}`;
+    this.name = 'LoaderError';
+    this.nativeError = error;
```

Comment:
Is `nativeError` somehow a common way of naming the wrapped error?

Reply:
It's not common, but as mentioned in the pull request description I utilised the same naming convention as React's `SyntheticEvent#nativeEvent` (https://reactjs.org/docs/events.html). I thought this was apt since the way the error is wrapped is similar to a synthetic event.

I did consider other implementations, but felt they weren't as clean. For instance, we could add a plain `meta` object to the error with some key that denotes the error is from `react-loosely-lazy`. But then this could have conflicts with existing errors using the same `meta` key from other libraries, so we'd need to make sure that we don't overwrite any existing `meta` object. The key would also need to be quite unique, which could be done just by adding the library name or using a `Symbol`, though symbols do not have full browser support. We could skip wrapping in something like `meta` altogether and just add in the unique key to the error object directly. Though either of these approaches will not play as well with type systems when accessing the unique keys, unless we expose the wrapped error type. But even then I don't think it would work that well.

If you'd prefer a different naming convention or pattern, then please let me know.